specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2018-09-01/iotcentral.json

```diff
@@ -733,6 +814,60 @@
           "readOnly": true
         }
       }
+    },
+    "AppTemplate": {
```

Comment:
>AppTemplate [](start = 5, length = 11)

The shape of this resource doesn't actually confirm to the RPC. Take a look at [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#get-resource](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#get-resource). 

Your resource should look like this example:
```
{
  "name": "{templateName}",
  "type": "Microsoft.IoTCentral/applicationTemplates",
  "id": "/subscriptions/{subscriptionId}/providers/Microsoft.IoTCentral/applicationTemplates/{templateName},
  "properties": {
     <this is where your custom properties go>
  }
}
```